core/App/utils/helpers.ts

```diff
@@ -28,9 +28,28 @@ export function hashCode(s: string): number {
   return s.split('').reduce((hash, char) => char.charCodeAt(0) + ((hash << 5) - hash), 0)
 }
 
-export function hashToRGBA(i: number) {
-  const colour = (i & 0x00ffffff).toString(16).toUpperCase()
-  return '#' + '00000'.substring(0, 6 - colour.length) + colour
+/**
+ * seeded random number generator between 0-1
+ * Adapted from https://stackoverflow.com/questions/521295/seeding-the-random-number-generator-in-javascript
+ */
+function mulberry32(a: number) {
```

Comment:
Are we able to use the fat arrow fn notation. I think its better to think of them as standard and `function` as alternative if you want to `this` scoped differently.